### PR TITLE
Shell scrip to run docker container

### LINKED ISSUE #96: Utility to run ASD on docker container
How to run docker container currently.

```
docker run -v "$(pwd)/work:/asd" -dit --init --rm --name asd ghcr.io/koriym/app-state-diagram:latest composer --quiet global exec asd /asd/profile.json
```

This is redundant. It should be changed as follows.

```
asd work/profile.json
```

### PR DESCRIPTION
**Pull request by koriym**
Closes #96 

# Installation and usage is now easier

Pull the docker image and install ASD utility.

```bash
docker pull ghcr.io/koriym/app-state-diagram:latest
```

```bash
curl -L https://raw.githubusercontent.com/koriym/app-state-diagram/32d45ca8c9a4d5bda0260596f947cb3cbba274a6/bin/asd.sh > /usr/local/bin/asd
chmod +x /usr/local/bin/asd
```
Perform the following steps to confirm.

```
mkdir work
curl -L curl https://koriym.github.io/app-state-diagram/blog/profile.json > work/profile.json
asd --watch ./work/profile.json
````
Has the ASD document been opened?

### CODE REVIEW

**Comment by koriym** [comment]
@apple-x-co レビューお願いします！

**Comment by koriym** [comment]
@mamund Isn't that amazing progress?

File: docker/app-state-diagram/run.sh [hunk 1]
```diff
@@ -1,4 +1,3 @@
 #!/usr/bin/env bash
 
-dir=${1:-$(pwd)/work}
-docker run -v "$(pwd)/work:/work" -it --init --rm --name asd -p 3000:3000 app-state-diagram:latest composer global exec asd -- --watch /work/profile.xml
\ No newline at end of file
+../../bin/asd.sh --watch "$(pwd)"/work/profile.xml
```
Comment (NaokiTsuchiya) [inline docker/app-state-diagram/run.sh:3]:
```suggestion
../../docs/asd.sh --watch "$(pwd)"/work/profile.xml
```

File: README.md [hunk 2]
```diff
@@ -32,31 +32,60 @@ You can try ASD in a minute without installing any tools.
...
 
-Watch mode
-```bash
-docker run -v "$(pwd)/work:/asd" -it --init --rm --name asd ghcr.io/koriym/app-state-diagram:latest composer global exec asd -- --watch /asd/profile.json
+Perform the following steps.
+
+```
+mkdir work
+curl -L curl https://koriym.github.io/app-state-diagram/blog/profile.json > work/profile.json
+asd --watch ./work/profile.json
```
Comment (NaokiTsuchiya) [inline README.md:54]: うまく動かないと思ったら composer/bin/asd にPATH通ってました 😅

**Review by NaokiTsuchiya — CHANGES_REQUESTED** [review]

**Review by NaokiTsuchiya — APPROVED** [review]
👍

**Pull request merged**